### ISSUE TITLE
Issues with labels are not being imported

### ISSUE THREAD

**Issue by anarute** [maubot/gitlab#73]
The errors on the logs are:

```python
Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/mautrix/types/util/serializable_attrs.py", line 212, in _dict_to_attrs
    obj = attrs_type(**new_items)
          ^^^^^^^^^^^^^^^^^^^^^^^
TypeError: GitlabLabel.__init__() missing 1 required positional argument: 'lock_on_merge'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/data/plugins/xyz.maubot.gitlab-v0.2.1.mbp/gitlab_matrix/webhook.py", line 117, in try_process_hook
    await self.process_hook(body, evt_type, room_id)
  File "/data/plugins/xyz.maubot.gitlab-v0.2.1.mbp/gitlab_matrix/webhook.py", line 130, in process_hook
    evt = EventParse[evt_type].deserialize(body)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/mautrix/types/util/serializable_attrs.py", line 375, in deserialize
    return _dict_to_attrs(cls, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/mautrix/types/util/serializable_attrs.py", line 198, in _dict_to_attrs
    new_items[name] = _try_deserialize(field_meta, value)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/mautrix/types/util/serializable_attrs.py", line 229, in _try_deserialize
    return _deserialize(field.type, value, field.default)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/mautrix/types/util/serializable_attrs.py", line 267, in _deserialize
    return _dict_to_attrs(cls, value, default, default_if_empty=True)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/mautrix/types/util/serializable_attrs.py", line 198, in _dict_to_attrs
    new_items[name] = _try_deserialize(field_meta, value)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/mautrix/types/util/serializable_attrs.py", line 229, in _try_deserialize
    return _deserialize(field.type, value, field.default)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/mautrix/types/util/serializable_attrs.py", line 277, in _deserialize
    return _deserialize(args[0], value, default)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/mautrix/types/util/serializable_attrs.py", line 280, in _deserialize
    return [_deserialize(item_cls, item) for item in value]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/mautrix/types/util/serializable_attrs.py", line 280, in <listcomp>
    return [_deserialize(item_cls, item) for item in value]
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/mautrix/types/util/serializable_attrs.py", line 267, in _deserialize
    return _dict_to_attrs(cls, value, default, default_if_empty=True)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/mautrix/types/util/serializable_attrs.py", line 218, in _dict_to_attrs
    raise SerializerError(
mautrix.types.util.serializable.SerializerError: Missing value for required key lock_on_merge in GitlabLabel
```

This was caused after a recent Gitlab upgrade where the `lock_on_merge` column was removed: https://gitlab.com/gitlab-org/gitlab/-/merge_requests/145446#note_1851829125

**Comment by tulir** [comment]
Are you maybe running #69 instead of the master branch? (I merged #70 instead of #69 for a reason)

**Comment by anarute** [comment]
> Are you maybe running #69 instead of the master branch? (I merged #70 instead of #69 for a reason)

It's possible, sorry for the noise.
